multinet/multinet.py

```diff
@@ -2,7 +2,7 @@
 from graphql import graphql
 import json
 
-from flask import Blueprint, request
+from flask import Blueprint, request, Response
```

Comment:
Shouldn't need this if we use the tuple form (as shown in my comment below).

Reply:
Without the response function we get this error: `TypeError: The view function did not return a valid response. The function either returned None or ended without a return statement.`

Reply:
Ah, I thought you could send an empty response, but it's not allowed (so, I'm not actually sure why wrapping `None` in `Response` works, but it probably will causes headaches down the line).

Instead of `None`, passing an empty string actually does work, so let's go with that.

Reply:
Fixed in the later commit.